src/common/components/contactInfo/contactInfo.component.js

```diff
@@ -44,7 +44,8 @@ class Step1Controller {
 
     this.requestRadioStation = !!(this.radioStationApiUrl && this.radioStationRadius)
 
-    this.radioStation = undefined
+    this.radioStationName = undefined
+    this.radioStationCallLetters = undefined
```

Comment:
I was thinking it would be better to explicitly declare an initial value of `undefined` for these values... but it doesn't seem like this is needed.  They will resolve to undefined anyway, until they're given values from the other methods.

Reply:
I think they should be removed.